src/Grid/utils.ts

```diff
@@ -105,25 +82,98 @@ export function getBreakpointKey(
 }
 
 /*
-    Using the passed responsiveSettings, determines what responsive value should be returned for
-    the current breakpoint. Ex:
+    CSS used both by generateBreakpointCSS() and the Cell component. Aligns content vertically.
+*/
+export const middleCSS = css`
+    display: inline-flex;
+    flex-flow: column wrap;
+    justify-content: center;
+    justify-self: stretch;
+`;
 
-    const settings = { xs: 1, sm: 3, md: 6 }
-    const innerWidth = 800;
-    const sortedBreakpoints = [{ xs: 500, sm: 750, md: 1000 }];
-    getResponsiveValue(settings, innerWidth, sortedBreakpoints); // 3
+/*
+    Small helper object for the generateBreakpointCSS() function to render the correct CSS for each
+    grid setting.
 */
-export function getResponsiveValue(
-    responsiveSettings: object | number | undefined,
-    innerWidth: number,
-    sortedBreakpoints: BreakpointType[]
+const ops = {
+    // If width is 0, don't show the cell. If middle is true, add in the middle CSS.
+    width: (w: any, middle: boolean) =>
```

Comment:
This is probably copied code, but we should never self-minify vars. Unless we're coding in go where that's a language preference, we should be verbose because our peers will benefit from the verbosity and our build tools will transpile this down to single-letter vars.

Reply:
Actually not copied. It was a small utility object/methods so I was continuing to keep it small. Point taken though, I'm normally a fan of variable verbosity.